### ISSUE TITLE
Tool focus - in Firefox focus is on more than one tool until user tabs all the way around

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MassGIS/morisoliver#110]
```
Josh Rosenthal of PeopleGIS explained some of the look of OLIVER.  The focus is 
indicated usually by a dashed outline. The darker blue is an indication of an 
active tool in a toogle set.  This is different concept than the tool with 
focus.  The tool with focus will get activated when a select function is used 
(for example, Enter key is pressed).  When you tab you don't want to activate 
the action immediately.  

Firefox and IE both show dashed outline - Chrome and Safari show nothing.  
Browsers provide ways of overriding way it looks - dashed outline is the most 
consistent.  Override options are limited.   Only Firefox (testing in 9.0.1) 
shows the issue with the dashed line around zoomin "sticking" until the user 
tabs all the way "around" the application and back to zoomin.  This could be a 
Mozilla bug that would be found in their Bugzilla.  This could be either very 
easy or very hard to fix.  A few minutes of looking into it did not indicate 
which.  Something strange going on with tab order - has to do with what Firefox 
is allowing to be an initial focused element - what selected element should be 
- able to remove selection from it.  Not explicitly telling it to look selected 
- allowing 2 elements to be selecting - half a bug and half an option to 
designate something as selectable. 

We decided to not pursue a fix for this, just to document it here.   
```

Original issue reported on code.google.com by `Aleda.Fr...@state.ma.us` on 9 Jan 2012 at 7:03
